Architecture/multiple-ovms-json-config.md

```diff
@@ -0,0 +1,115 @@
+# Multiple OpenVINO Model Server Config JSON
+
+<!--ts-->
+
+- [Status](#status)
+- [Decision](#decision)
+- [Context](#context)
+- [Proposed Design](#proposed-design)
+- [Consequences](#consequences)
+- [References](#references)
+
+<!--te-->
+
+## Decision
+
+<!-- Requirements approval board will update this section with justification for approval or rejection -->
+
+## Context  
+
+<!-- Please provide context to the requirement. -->
+
+Currently, we use same config.json for all instances of OpenVINO Model Server(OVMS) pipelines, which leads to some issue regarding the device mounting for OVMS server: see issue intel-retail/automated-self-checkout#322. So we need a way to have multiple or unique config json file per OVMS instance.
+
+## Proposed Design 
+
+<!-- Please provide a high level design of the proposed requirement. -->
+
+Move the current device update logic from `run_ovms.sh` with the config.json file into profile-launcher in Golang. When profile-launcher about to launch a new instance of OVMS server, it then produces a unique config json file name for that instance of OVMS server. 
```

Comment:
`run_ovms.sh` has been updated with just `run.sh`

Reply:
good catch, and updated.